release: v0.12.0rc (#1210)
rero/rero-ils: RELEASE-NOTES.rst

   values.
-  Allows (temporarily) to attached to a document of journal type or
   periodical issuance type either a serial holdings or a standard
   holdings. This is necessary to migrate all the legacy system records
   to RERO ILS.

Acquisition
~~~~~~~~~~~

-  Updates the document and item JSON schemas for the new acquisition
   list management. These lists are generated through an ES query that
   filters the newly acquired items with a specific time span. This
   allows a librarian to define a permalink to be shared through the
   library website (or elsewhere), that points to a RERO ILS search
   result presenting the list.
-  Adds a search input in the order line editor to find a specific
   document and to save the librarian the burden to type the full REST
   API document URL.

API
~~~

-  Adds ``marcxml`` support to the document API, thus allowing
   authenticated user to post ``marcxml`` records using an external
   script. The record is added to the database with the ``_draft`` flag
   set to true, to disable the validation of the data and to avoid the
   record to be found in the catalog.

Permission
~~~~~~~~~~

> -  Adds the ``document_importer`` role for user that post records

Reviewer: ```suggestion
-  Adds the ``document_importer`` role to users posting records
```
